src/scipp/html/formatting_html.py

```diff
@@ -224,21 +224,15 @@ def summarize_attrs(attrs, embedded_in=None):
     return f"<ul class='sc-var-list'>{attrs_li}</ul>"
 
 
-def find_bin_edges(var, ds):
+def _find_bin_edges(var, ds):
     """
     Checks if the coordinate contains bin-edges.
     """
-    bin_edges = []
-    for idx, dim in enumerate(var.dims):
-        length = var.shape[idx]
-        if not ds.dims:
-            # Have a scalar slice.
-            # Cannot match dims, just assume length 2 attributes are bin-edge
-            if length == 2:
-                bin_edges.append(dim)
-        elif dim in ds.dims and ds.shape[ds.dims.index(dim)] + 1 == length:
-            bin_edges.append(dim)
-    return bin_edges
+    return [
+        dim
+        for length, dim in zip(var.shape, var.dims)
```

Comment:
```suggestion
        for length, dim in var.sizes.items()
```
or am I missing the point here?